components/server/src/workspace/gitpod-server-impl.ts

```diff
@@ -2034,39 +2035,7 @@ export class GitpodServerImpl implements GitpodServerWithTracing, Disposable {
         const desc = await client.describeWorkspace(ctx, req);
         const duration = desc.getStatus()!.getSpec()!.getTimeout();
 
-        return { duration, canChange, humanReadableDuration: this.goDurationToHumanReadable(duration) };
-    }
-
-    goDurationToHumanReadable(goDuration: string): string {
-        const [, value, unit] = goDuration.match(/^(\d+)([mh])$/)!;
-        let duration = parseInt(value);
-
-        switch (unit) {
-            case "m":
-                duration *= 60;
-                break;
-            case "h":
-                duration *= 60 * 60;
-                break;
-        }
-
-        const hours = Math.floor(duration / 3600);
-        duration %= 3600;
-        const minutes = Math.floor(duration / 60);
-        duration %= 60;
-
-        let result = "";
-        if (hours) {
-            result += `${hours} hour${hours === 1 ? "" : "s"}`;
-            if (minutes) {
-                result += " and ";
```

Comment:
In the new implementation, there's a slight change where the `and` is omitted for simplicity. Should we want it, we can easily adjust the way we join time segments and include an `and`